tox.ini

```diff
@@ -1,7 +1,7 @@
 [flake8]
 count = True
 #default_ignorelist: E121,E123,E126,E226,E24,E704,W503,W504
-ignore = U101, SCS109, SCS101, SCS113
+ignore = U101, SCS109, SCS101, SCS113, W503
```

Comment:
Yeah we should either disable W503 or W504, as they are kinda in direct conflict with each other (they describe each others best practice as antipattern and vice versa)

https://www.flake8rules.com/rules/W503.html
https://www.flake8rules.com/rules/W504.html

Not sure if I have a very strong preference tbh